app/kubernetes/views/deploy/deployController.js

```diff
@@ -117,7 +118,16 @@ class KubernetesDeployController {
   }
 
   setStackName(name) {
-    this.formValues.StackName = name;
+    return this.$async(async () => {
+      if (KUBE_STACK_NAME_VALIDATION_REGEX.test(name) || name === '') {
+        this.state.stackNameError = '';
+      } else {
+        this.state.stackNameError =
+          "Stack must consist of alphanumeric characters, '-', '_' or '.', must start and end with an alphanumeric character and must be 63 characters or less (e.g. 'my-name', or 'abc-123').";
```

Comment:
Add missing `stackNameError: '',` in state definition L61
```js
this.state = {
  ....
  isEditorReadOnly: false,
  selectedHelmChart: '',
  stackNameError: '', // here
}
```

Reply:
Thanks for the spot! Updated